LSP.sublime-settings

```diff
@@ -111,8 +111,12 @@
   // Show symbol action links in hover popup if available
   "show_symbol_action_links": false,
 
-  // Disable Sublime Text's explicit and word completion.
-  "only_show_lsp_completions": false,
+  // Disable Sublime Text's snippet completions.
+  "inhibit_explicit_completions": false,
```

Comment:
Could we give it a more understandable name than "explicit"? I know that ST calls it that but it's not really clear what it means and we could just give it a better and clearer name - namely `inhibit_snippet_completions`.